packages/utils/src/route.js

```diff
@@ -56,7 +56,11 @@ function cleanChildrenRoutes (routes, isChild = false, routeNameSplitter = '-',
     route.path = isChild ? route.path.replace('/', '') : route.path
     if (route.path.includes('?')) {
       if (route.name.endsWith(`${routeNameSplitter}index`)) {
-        route.path = route.path.replace(/\?$/, '')
+        if (trailingSlash) {
+          route.path = route.path.replace(/\?\/$/, '/')
```

Comment:
The change looks good.

Just double check, will this affect logic inside below route.path.split(/) ?

Reply:
No, it dosn't.

before change:
```js
[
  ':child?',
   ''
]
```

after change:
```js
[
  ':child',
  ''
]
```

it just removes `?` same as without `trailingSlash` option